common/src/main/java/tachyon/Constants.java

```diff
@@ -248,4 +248,7 @@
   public static final String TACHYON_SECURITY_AUTHENTICATION = "tachyon.security.authentication";
   public static final String TACHYON_AUTHENTICATION_PROVIDER_CUSTOM_CLASS =
       "tachyon.authentication.provider.custom.class";
+  // Authorization
+  public static final String FS_PERMISSIONS_UMASK_KEY = "tfs.permission.umask";
```

Comment:
Can we have this property string consistent with other Tachyon properties, maybe something like
"tachyon.fs.permission.umask"?